### ISSUE TITLE
wrong dark background with dark letters on amazon-sites

### ISSUE THREAD

**Issue by Dan-cer** [horst3180/Vertex-theme#110]
Nearly all is fine with this excellent theme, thank You. So far only on amazon-sites there are some unreadable frames:
1. E-Books in Kindle Cloud Reader, see uploaded image no. 1
2. Frames with Description of an offered book (or other), see uploaded image no. 2
Is there a solution to deal with these amazon-codes?
![cloud-reader](https://cloud.githubusercontent.com/assets/800638/7988665/e3b86c12-0ae4-11e5-927f-079dae7f8c6a.png)
![amazon-buch](https://cloud.githubusercontent.com/assets/800638/7988670/ea4ee128-0ae4-11e5-80fa-bd22be64c544.png)

**Comment by horst3180** [comment]
I'll have a look at this. It should be possible to fix.

**Comment by Dan-cer** [comment]
Great! Good news!
Thank You! I love Your theme!

Am 10.06.2015 um 18:37 schrieb horst3180:

**Comment by horst3180** [comment]
I think commit f10538f9dc0c9421162cc33747dbea3e0314b168 makes it a bit better, but I don't think it's completely fixed. Somehow the about:blank page background is coming through on these amazon sites. Very weird...

**Comment by Dan-cer** [comment]
Yes, here it is nearly fine with amazon and kindle cloud reader. white 
background, black fonts.

Am 10.06.2015 um 22:03 schrieb horst3180:
>
> I think commit f10538f 
> <https://github.com/horst3180/Vertex-theme/commit/f10538f9dc0c9421162cc33747dbea3e0314b168> 
> makes it a bit better, but I don't think it's completely fixed. 
> Somehow the about:blank page background is coming through on these 
> amazon sites. Very weird...
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/horst3180/Vertex-theme/issues/110#issuecomment-110894450>.
>

-- 
best regards
Rainer

**Comment by exten** [comment]
I want to help you, but I have limited skills.. As long as you persist, you will succeed!






At 2015-06-11 04:03:29, "horst3180" <notifications@github.com> wrote:


I think commit f10538f makes it a bit better, but I don't think it's completely fixed. Somehow the about:blank page background is coming through on these amazon sites. Very weird...

**Comment by Dan-cer** [comment]
I found that still there are unreadable sites: For example "about:config". See attached image.
![about-config](https://cloud.githubusercontent.com/assets/800638/8149687/cc5b7876-12cc-11e5-84c8-efab4ce5cb55.png)

**Comment by exten** [comment]
As high as "A" in the status bar at the possibility of designing and B it? "B" label height to reduce the possibility of that? I think so, people can focus more on reading data content.






At 2015-06-11 04:03:29, "horst3180" <notifications@github.com> wrote:


I think commit f10538f makes it a bit better, but I don't think it's completely fixed. Somehow the about:blank page background is coming through on these amazon sites. Very weird...

**Comment by Dan-cer** [comment]
I'm using already f10538f (https://github.com/horst3180/Vertex-theme/commit/f10538f9dc0c9421162cc33747dbea3e0314b168). The "about:config"-Site is  WITH the changed userContent.css

**Comment by horst3180** [comment]
I can't reproduce the about:config issue. Do you have any extensions installed that could interfere with the theme? In the screenshot the active tab doesn't look right either.

**Comment by Dan-cer** [comment]
I deactivated all addons and activated the default firefox theme. But it 
is the same as U can see in attached image.

Am 20.06.2015 um 14:10 schrieb horst3180:
>
> I can't reproduce the about:config issue. Do you have any extensions 
> installed that could interfere with the theme? In the screenshot the 
> active tab doesn't look right either.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/horst3180/Vertex-theme/issues/110#issuecomment-113753907>.
>

-- 
mit freundlichen Grüßen
Rainer

**Comment by Dan-cer** [comment]
With the newest download-zip-file and the included new userContent.css all is working fine! The "about:config"-site is fine, too!!! Thank You.
This issue is solved.

**Comment by Dan-cer** [comment]
A new issue appeared with a LastPass site, opened in Firefox, as You can see in the attached image. The text is nearly unreadable - white on white background.
In Chromium.browser the site is displayed normally = OK.
![lastpass](https://cloud.githubusercontent.com/assets/800638/8439632/873f85a4-1f6d-11e5-93f5-db06e9288c9f.png)
